crates/biome_formatter/src/format_element.rs

```diff
@@ -232,6 +232,10 @@ impl FormatElement {
     pub const fn is_space(&self) -> bool {
         matches!(self, FormatElement::Space)
     }
+
+    pub const fn is_line(&self) -> bool {
+        matches!(self, FormatElement::Line(_))
+    }
```

Comment:
Kinda surprised this didn't exist before. Turns out it's not actually necessary because `may_directly_break` does this check itself, but I figure this makes sense to keep around anyway.